modules/conflict_resolver/jsx/conflict_resolver.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react';
+import {Tabs, TabPane} from 'Tabs';
+import UnresolvedFilterableDataTable from './unresolved_filterabledatatable';
+import ResolvedFilterableDataTable from './resolved_filterabledatatable';
+
+class ConflictResolver extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      activeTab: 'unresolved',
+    };
+
+    this.onTabChange = this.onTabChange.bind(this);
+  }
+
+  onTabChange(tabId) {
+    this.setState({activeTab: tabId});
+  }
+
+  render() {
+    const tabs = [
+      {id: 'unresolved', label: 'Unresolved'},
+      {id: 'resolved', label: 'Resolved'},
+    ];
+
+    let filtertable;
+    switch (this.state.activeTab) {
+      case 'unresolved':
+        filtertable = (
+          <UnresolvedFilterableDataTable />
+        );
```

Comment:
just a suggestion, but when jsx is assigned to variables on one line, the parentheses aren't necessary!
e.g. `filtertable = <UnresolvedFilterableDataTable />;`